waku/v2/node/wakunode2.go

```diff
@@ -246,9 +247,9 @@ func New(opts ...WakuNodeOption) (*WakuNode, error) {
 	// Setup peer connection strategy
 	cacheSize := 600
 	rngSrc := rand.NewSource(rand.Int63())
-	minBackoff, maxBackoff := time.Second*30, time.Hour
+	minBackoff, maxBackoff := time.Minute, time.Hour
```

Comment:
Improvement: We should start moving these values to config, so that it helps in tuning these params while testing without code changes. We can have a default config read from a file, so that tuning can be done there. Will take it up as part of peer management enhancements.